racunaljka.js

```diff
@@ -0,0 +1,230 @@
+/*=================================== brojevi==================================================*/
+
+function seeAll() {
+document.getElementById('izvestaj').focus();
+}
+
+function brojJedan() {
+    var pise = document.getElementById('izvestaj');
+   pise.value += 1;
+}
+
+function brojDva() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 2;
+}
+
+function brojTri() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 3; 
+}
+
+function brojCetiri() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 4; 
+}
+
+function brojPet() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 5; 
+}
+
+function brojSest() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 6; 
+}
+
+function brojSedam() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 7; 
+}
+
+function brojOsam() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 8; 
+}
+
+function brojDevet() {
+    var pise = document.getElementById('izvestaj');
+    pise.value += 9; 
+}
+
+function brojNula() {
+    var pise = document.getElementById('izvestaj');
+    if(pise.value==="") {
+       pise.value='';
+    }
+    else{pise.value+=0}
+}
```

Comment:
The solution where you have a separate function for each button which you manually type does not seem scalable -- for example imagine that you have a calculator which has number from 1 to 100, would you write 100 functions? How would you maintain and improve code like that -- very hard!

There are multiple ways to do this in a more efficient way, here are some hints:
1) All methods are the same, just the number is different
2) Check how function binding works
3) You can iterate over all buttons via query and then use function binding